junit-platform-console/junit-platform-console.gradle

```diff
@@ -61,6 +69,27 @@ artifacts {
 	archives shadowJar
 }
 
-distZip.dependsOn(shadowJar)
-distTar.dependsOn(shadowJar)
-installDist.dependsOn(shadowJar)
+task runStandaloneConsole(type: JavaExec) {
+	workingDir "$buildDir/libs"
+	main = '-jar'
+	args = [
+			"junit-console-$rootProject.version-standalone.jar",
+			'--scan-classpath',
+			'--details', 'none'
```

Comment:
Again it's not really clear to me why these are indented with 2 relative tabs rather than 1.